.github/workflows/ci.yml

```diff
@@ -1,123 +0,0 @@
-name: CI
```

Comment:
is this removal of workflow intentional?

Reply:
Yes. It was removed since we're using GitHub actions here: https://github.com/ember-fastboot/ember-cli-fastboot/pull/832#discussion_r645785552

Reply:
No! Fixing it! Thanks.